utils/date.tsx

```diff
@@ -1,13 +1,24 @@
-import {format, parseISO} from 'date-fns';
+import {format} from 'date-fns';
 
-export const utcDateToLocalTimeString = (dateString: string | undefined): string => {
-  if (!dateString) {
+const MISSING_TIMEZONE = /^\d{4}-\d\d-\d\dT\d\d:\d\d:\d\d(\.\d+)?$/;
+
+// Some avalanche.org endpoints return dates without timezone information! This method is used to detect and fix them, by assuming they're UTC.
+export const fixMalformedISO8601DateString = (date: string) => (MISSING_TIMEZONE.test(date) ? `${date}Z` : date);
```

Comment:
I tried to be very conservative here: the input has to match a specific date and time format and not have any timezone information at the end. Otherwise, we just return the input string. There's a bunch of tests but LMK if there are others you think should be in there.